cli/dcoscli/cluster/main.py

```diff
@@ -0,0 +1,214 @@
+"""Provides status about the DCOS cluster managed by this CLI
+
+Usage:
+   dcos cluster --help
+   dcos cluster --info
+   dcos cluster --version
+   dcos cluster help
+   dcos cluster status [--json]
+
+Options:
+    --help                  Show this screen
+    --info                  Show info
+    --json                  Print json-formatted cluster status
+    --version               Show version
+"""
+import dcoscli
+import docopt
+from dcos import cmds, emitting, util
+from dcos.errors import DCOSException
+from dcos.http import request
+from dcoscli import tables
+
+logger = util.get_logger(__name__)
+
+emitter = emitting.FlatEmitter()
+
+MESOS_STATE_URL = '{}:5050/state.json'
```

Comment:
See the functions in `mesos.py` for accessing mesos.  Specifically, the `MasterClient` class.